prep_local

```diff
@@ -10,7 +10,7 @@ set -o errexit -o nounset -o pipefail
 : ${VIRTUAL_ENV?"Must be run in a python virtual environment"}
 
 # Install the latest version of pip
-pip install pip==8.1.2
+pip install --upgrade pip
```

Comment:
This is just looks like an ugly "pip" issue, that got fixed from 8.1.1 to 8.1.2

https://github.com/pypa/pip/commit/8e236dd6a09bd2f70f9d4fc886da8c354d4c58f2#diff-297395da5417032dc5d0738a1fc7e136L1002

For pip itself, I think.

`pip install --upgrade pip`

is a reasonable solution.

For other packages, we will require the hardcoded version.

So, this change looks good to me.

If you want to be conservative, remove the `pip install pip==8.1.2` line itself and we can upgrade pip in the buildinfo.json.  
Since this is a bug in pip, these are the two only two possible options for us.